test/e2e/db_cluster_parameter_group.py

```diff
@@ -13,7 +13,47 @@
 
 """Utilities for working with DB cluster parameter group resources"""
 
+import datetime
+import time
+
 import boto3
+import pytest
+
+DEFAULT_WAIT_UNTIL_DELETED_TIMEOUT_SECONDS = 60*10
+DEFAULT_WAIT_UNTIL_DELETED_INTERVAL_SECONDS = 15
+
+
+def wait_until_deleted(
+        cpg_name: str,
```

Comment:
nit: we could make this function take a `get` function parameter? and reuse it everywhere?

Reply:
copy paste from elsewhere... I can try to refactor in a later PR.